File: test/ocamlTests/BoolTest/xorTest.ml
test "xor(true,true)" (fun () -> expect (Bool.xor true true) |> toEqual Eq.bool false) ; 
test "xor(true,false)" (fun () -> expect (Bool.xor true false) |> toEqual Eq.bool true) ; 
test "xor(false,true)" (fun () -> expect (Bool.xor false true) |> toEqual Eq.bool true) ; 
test "xor(false,false)" (fun () -> expect (Bool.xor false false) |> toEqual Eq.bool false) ; 

File: test/fsharpTests/BoolTest/xorTest.fs
testCase "xor(true,true)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Bool.xor true true) "error"
testCase "xor(true,false)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Bool.xor true false) "error"
testCase "xor(false,true)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Bool.xor false true) "error"
testCase "xor(false,false)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Bool.xor false false) "error"
